src/components/stageLayout/StageLayout.styles.scss

```
@@ -147,8 +147,11 @@
 		.stageLayout__content {
 			justify-content: flex-start;
 			align-items: center;
-			margin-top: 48px !important;
-			padding-top: $grid-base-four;
+			// The sticky step header brings its own spacing; anything extra
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)